packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.ts

```diff
@@ -1,6 +1,7 @@
 import { ManfredAwesomicCV } from '@/model';
-import { ProfileSectionVm, RelevantLinkVm, ManfredRelevantLink } from './profile-section.vm';
+import { ProfileSectionVm, RelevantLinkVm, ManfredRelevantLink, PhoneNumbers } from './profile-section.vm';
 
+//Sólo afecta al .json de David Bonilla
```

Comment:
Please in english and ellaborate about the issue

Reply:
it's fixed